products/bash/release-ace-integration-server.sh

```diff
@@ -70,6 +76,15 @@ else
     tracing_namespace=${namespace}
 fi
 
+# ------------------------------------------------ SET ACE POLICY --------------------------------------------------
+
+if [[ "$eei_config" == "true" ]]; then
+  ace_policy_names="[ace-policyproject-eei]"
+else
+  ace_policy_names="[ace-keystore, ace-policyproject-ddd, ace-serverconf, ace-setdbparms, application.kdb, application.sth, application.jks]"
+fi
+echo -e "\nINFO: ACE policy configuration is set to: '$ace_policy_names'"
```

Comment:
Interesting way of configuring the policy names. Maybe rather than using a flag to decide whether to use EEI or DDD policy names a parameter could be used to pass the list in? And default to the DDD list? That way it would work for other demos in the future?

Reply:
Done